ProjFS.Mac/PrjFSKext/KauthHandler.cpp

```diff
@@ -637,6 +669,7 @@ KEXT_STATIC int HandleFileOpOperation(
                 currentVnode,
                 vnodeFsidInode,
                 path,
+                nullptr, // TODO
```

Comment:
what does the TODO mean?

Reply:
Leftovers from the first iteration of this PR, when we still had the fully flexible arbitrary strings.